aio/src/styles/1-layouts/_top-menu.scss

```
@@ -244,6 +244,7 @@ aio-search-box.search-container {
   .toolbar-external-icons-container {
     display: flex;
     flex-direction: row;
+    flex-shrink: 0;
```

Could you add a comment here to avoid someone accidentally removing it in the future (thinking it is not needed):

```suggestion
    flex-shrink: 0;  // This is required for the icons to be displayed correctly in IE11.
```